OpenRA.Game/GameRules/Ruleset.cs

```diff
@@ -261,5 +267,14 @@ static bool AnyFlaggedTraits(ModData modData, List<MiniYamlNode> actors)
 
 			return false;
 		}
+
+		/// <summary>
+		/// Filter for concrete Nodes.
+		/// Abstract nodes (e.g. rooted from support directory) are not accepted.
+		/// </summary>
+		private static bool IsConcreteNode(MiniYamlNode node)
+		{
+			return !SymbolicSupportDirectory.IsPathRelativeToSupportDirectory(node.Key);
```

Comment:
The `^` convention for actor names is completely separate to the convention for using `^` to denote the support directory in file paths.  Mixing the two here isn't a good idea!

Reply:
@pchote - Yeah, in hindsight I should have just gone with a minimal change on the impl. 

How would you feel about definition of "abstractActorPrefix" string constant on ActorInfo to disambiguate that case, vs dropping the constants business and sticking with the string literals?